### PR TITLE
mktemp: add page

### PR DESCRIPTION
**Pull request by jedahan**

### CODE REVIEW

File: pages/common/mktemp.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mktemp
+
+> make a unique temporary file name
```
Comment (agnivade) [inline pages/common/mktemp.md]: Suggestion-> "make a temporary file with a unique name"

**Comment by waldyrious** [comment]
@jedahan you need to add the email set in your local git config (`git config user.email`) to your github account (https://github.com/settings/emails) so that the commit can be properly attributed to you.

**Comment by agnivade** [comment]
@jedahan - Any updates on this ?

**Comment by waldyrious** [comment]
@agnivade the CLA has already been signed by @jedahan under the email he has associated to his github account, so we could edit this branch using git filter-branch to change the email address associated with his commit.

The feature to enable editing of the original branch of pull-requests by the upstream maintainers was [announced in September 2016](https://github.com/blog/2247-improving-collaboration-with-forks), so I believe we could [directly edit his branch](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/), without relying on [these tricks](https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally) as we had to before.

**Comment by waldyrious** [comment]
@agnivade the CLA has already been signed by @jedahan under the email he has associated to his github account, so we could edit this branch using git filter-branch to change the email address associated with his commit.

The feature to enable editing of the original branch of pull-requests by the upstream maintainers was [announced in September 2016](https://github.com/blog/2247-improving-collaboration-with-forks), so I believe we could [directly edit his branch](https://help.github.com/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork/), without relying on [these tricks](https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally) as we had to before.

**Comment by waldyrious** [comment]
@jedahan why did you close this?

**Comment by jedahan** [comment]
That was an accident!

**Comment by waldyrious** [comment]
For future reference: the remote branch of this PR is gone ("jedahan:master was force-pushed and no longer has any new commits.") so the contents are likely unrecovarable at this point. Hopefully someone will submit a page for this command again.

**Pull request closed without merging**